packages/manager/modules/sms/src/sms/batches/history/delete/delete.routing.js

```diff
@@ -0,0 +1,33 @@
+import controller from './delete.controller';
+import template from './delete.html';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('sms.service.batches.history.delete', {
+    url: '/delete?batchId&batchName&serviceName',
+    views: {
+      modal: {
+        // controller,
+        // template,
```

Comment:
To remove

Reply:
Comment on a file that should be part of the release, sorry!